### ISSUE TITLE
Redridge Drudger

### ISSUE THREAD

**Issue by Phatcat** [elysium-project/server#1783]
Is dual wielding weapons, but only hits with his main hand, and seems to be hitting too fast.. shouldn't he be hitting with more normal speed, but using both his weapons?

**Comment by Phatcat** [comment]
Yeah, so I looked him up in Bestiary, and yes, he should indeed known dual wield;
![billede](https://user-images.githubusercontent.com/1907820/28995456-c5d5fd24-79e9-11e7-8a09-bdbd06a8c962.png)

**Comment by kampeador** [comment]
I had some time and looked around this issue.
It seems it is not related to the Redridge Drudger or dual wielding directly.
Currently, it depends on the NPC equipment and it is messed up.
For example:
One of his items id is 2558. with subclass id: 4(One handed mace).
If you change that class to 5(Two handed mace). His attack ready animation will change to the two handed, second weapon will disappear, but he will use a dual wield, so gnolls like Riverpaw Brute are also bugged.
If you change that class to 10(Staff). His attack ready animation will disappear/bugged, but the attack animation itself will work correct.
You can test gnoll attack animations using .mod morph 384(or any other gnoll display id) and equip different weapons.

Some old emulators(pre wotlk) have changed creature emotes/animations manually depending, on the weapon type and other attributes. I've looked in the core and there is nothing like that here. 
I didn't have much time to research in this area, but it seems that some generic approach is used here, which may not work in this case.
